lzy/allocator/src/main/java/ai/lzy/allocator/util/DaoUtils.java

```diff
@@ -0,0 +1,20 @@
+package ai.lzy.allocator.util;
+
+public final class DaoUtils {
```

Comment:
you may with to move it to the `utils-db` module

Reply:
Moved it